src/renderer/components/MapEditor/ICCAExportDialog.js

```diff
@@ -0,0 +1,494 @@
+  question4Answer4: 'Other',
+  // Placeholder for question 5
+  question5Placeholder: 'Your contact information',
+  // Helper texts
+  requiredAnswer: 'This answer is required.',
+  atLeastOneAnswer: 'Please fill in at least one field.',
+  errorMissingRequired: 'Please fill out all required fields.',
+  // Labels for community name
+  communityOriginalName: 'Original language',
+  communityEnglishName: 'English'
+})
+
+// Insert a tiny lil' shake animation when an error occurs
+// We don't use the makeStyles api here because it can't handle keyframes
+insertCss(`
+  @keyframes shakeX {
+    from,
+    to {
+      transform: translate3d(0, 0, 0);
+    }
+
+    16.67%,
+    50%,
+    83.33% {
+      transform: translate3d(-10px, 0, 0);
+    }
+
+    33.33%,
+    66.67% {
+      transform: translate3d(10px, 0, 0);
+    }
+  }
+
+  .shakeX {
+    animation-name: shakeX;
+    animation-timing-function: ease-in-out;
+    animation-duration: 500ms;
+    animation-delay: 0;
+  }
+`)
+
+const EditDialogContent = ({ onClose }) => {
+  const classes = useStyles()
+  const { formatMessage } = useIntl()
+  const [saving, setSaving] = useState()
+  const [error, setError] = useState(false)
+  const [value1, setValue1] = useState('')
+  const [value2, setValue2] = useState('')
+  const [value3a, setValue3a] = useState('')
+  const [value3b, setValue3b] = useState('')
+  const [value4, setValue4] = useState('')
+  const [value5, setValue5] = useState('')
+  const [value1Error, setValue1Error] = useState(false)
+  const [value2Error, setValue2Error] = useState(false)
+  const [value3Error, setValue3Error] = useState(false)
+  const [value4Error, setValue4Error] = useState(false)
+  const [value5Error, setValue5Error] = useState(false)
+  const ref = useRef()
+
+  // Pre-optimize the parent element for the "shake" animation when there's
+  // an error. If we don't do this, the first time the animation plays,
+  // it feels janky
+  useEffect(() => {
+    if (!ref.current) return
+    ref.current.parentNode.style.willChange = 'transform'
+  })
+
+  const handleClose = () => {
+    setSaving(false)
+    onClose()
+  }
+
+  const handleSave = async e => {
+    e.preventDefault()
+
+    // Input validation
+    let error = false
+    if (value1 === '') {
+      setValue1Error(true)
+      error = true
+    }
+    if (value2 === '') {
+      setValue2Error(true)
+      error = true
+    }
+    if (value3a === '' && value3b === '') {
+      setValue3Error(true)
+      error = true
+    }
+    if (value4 === '') {
+      setValue4Error(true)
+      error = true
+    }
+    if (value5 === '') {
+      setValue5Error(true)
+      error = true
+    }
+
+    // Bail early if there's a required field missing
+    if (error === true) {
+      setError(true)
+      errorShake()
+      return
+    }
+
+    setSaving(true)
+    const points = await getGeoJson()
+    const metadata = {
+      communityConsent: value1,
+      makePublic: value2,
```

Comment:
Can we write value1 & 2 as boolean values?

Reply:
Yep. Unfortunately HTML values can only be strings, but we can convert that to boolean values when we write out the JSON.